drivers/counter/Kconfig.nrfx

```diff
@@ -112,6 +112,19 @@ config COUNTER_RTC0_PRESCALER
 	help
 	  Frequency = 32768 / (prescaler+1).
 
+config COUNTER_RTC0_GUARD_PERIOD_MS
```

Comment:
We have to include this guard in counter_api_get_max_relative_alarm.

Reply:
done